.github/workflows/build_ffmpeg.yaml

```diff
@@ -10,7 +10,9 @@
 name: Build non-GPL FFmpeg from source
 
 on:
-  workflow_dispatch:
+  pull_request:
```

Comment:
I think we still need the `workflow_dispatch` stuff for us to trigger this workflow manually

Reply:
Ah! I misread the docs. I thought we only needed that if we wanted to have inputs. Let's fix now.